### PR TITLE
Update nb.view.php

### PR DESCRIPTION
**Pull request by tipierre66**
line 29 <?php
what does it call ?

### CODE REVIEW

File: views/front/comment/nb.view.php [hunk 1]
```diff
@@ -26,5 +26,4 @@
     }
     ?>
     </div>
-    <?php
 }
```
Comment (sdrdis) [inline views/front/comment/nb.view.php:29]: Note: This is PHP code

**Comment by sdrdis** [comment]
hmmm... Thanks, but I am not sure I understood what you intended to do. The closing bracket is PHP code, so we need this `<?php` tag. Could you clarify ?

Thanks again

**Comment by Savageman** [comment]
@tipierre66 Still no news from you, should we close it?

**Comment by tipierre66** [comment]
yes it has to be closed

-------- Original message --------
Subject: Re: [noviusos_blognews] Update nb.view.php (#5)

**Pull request closed without merging**